tests/Doctrine/Tests/KeyValueStore/Storage/AmazonDynamoDbTest.php

```diff
@@ -0,0 +1,278 @@
+<?php
+
+namespace Doctrine\Tests\KeyValueStore\Storage;
+
+use Doctrine\KeyValueStore\Storage\AmazonDynamoDbStorage;
+
+class AmazonDynamoDbTest extends \PHPUnit_Framework_TestCase
```

Comment:
Add `@covers \Doctrine\KeyValueStore\Storage\AmazonDynamoDbStorage` to avoid coverage leakage